### ISSUE TITLE
When moving from a menu to another one the page become blank and then shows the sections

### ISSUE THREAD

**Issue by akramsarkis** [Geeks-Solutions/nuxt-sections#91]
- [ ] When moving from a menu to another one the page become blank and then shows the sections
- What is expected is to have charge the sections from the api which is the case but we should not see a transition like a blank page showing and then the sections charged

### **Acceptance testing**

- [ ] When moving from a page to another page on a site using sections, the sections should be as now loaded from the api but we must not see that there is a transition like now a blank page showing and then the sections loaded

**Comment by akramsarkis** [comment]
isn't this done @mahfouzhseineweev  ?

**Comment by mahfouzhseineweev** [comment]
@akramsarkis Yes, update was installed on meta sections and Maiori. You can QA it and confirm the problem is fixed to close the issue

**Comment by akramsarkis** [comment]
this issue is no more valid as it is already implemented, closing it
